src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String indentation = "    ";
```

Comment:
Hi! In this case since the indentations are constants, would it be better to name them as finals and in capital letters instead? e.g. "INDENTATION" 

((Hi I'm back to do more reviews))

Reply:
oh, thank you! I've corrected them. Hahhah